libraries/SITL/SIM_config.h

```diff
@@ -55,6 +55,10 @@
 #define AP_SIM_SLUNGPAYLOAD_ENABLED (CONFIG_HAL_BOARD == HAL_BOARD_SITL)
 #endif
 
+#ifndef AP_SIM_FLIGHTAXIS_ENABLED
```

Comment:
Pulls this out as a separate easy-to-merge PR?

Reply:
Yup, as mentioned, there is a precursor PR already open: #28282. This PR depends on it, so GitHub showing more in the diff than I want.

Filter to just the last two commits.